server/src/models/Expense.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+# Location is stored as an x and y coordinate (easier to input into maps APIs)
+# All fields are currently required
+class Expense(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    # .... the rest of the fields
+    class Config:
+        allow_population_by_field_name = True
+        arbitrary_types_allowed = True
+        json_encoders = {ObjectId: str}
```

Comment:
Move this to the bottom of the class. See this as an example: https://www.mongodb.com/developer/quickstart/python-quickstart-fastapi/#database-models